oracle-ebs-122/2.1/modules/ROOT/pages/new-reference.adoc

```diff
@@ -0,0 +1,503 @@
+= Oracle EBS 12.2 Connector Reference - Mule 4
+// :page-aliases: connectors::oracle/oracle-ebs-122-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Oracle EBS 12.2 Connector v2.1
```

Comment:
Delete line 6.

Reply:
I remember we added this on purpose in a previous release to clarify the version since 12.2 is in the name. But if it looks weird or doesn't match the template I can remove it-- what do you think? @valkyrie69